src/GSoft.Extensions.MediatR.Analyzers/Internals/RuleCategories.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace GSoft.Extensions.MediatR.Analyzers.Internals;
+
+internal static class RuleCategories
+{
+    // Same categories used by Microsoft
```

Comment:
So no constants from the Roslyn analyzers package for that? 😥

Reply:
No constants for that. The documentation says it's an arbitrary category. In the .NET runtime code, it's hardcoded. Example: https://github.com/dotnet/runtime/blob/ac5fe4f7f767d4fb04be85fc9de40cbe0f54d732/src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/DiagnosticDescriptors.cs#L15